pkg/object/objects.go

```diff
@@ -448,14 +448,15 @@ func buildResourceRegexp(s string) (*regexp.Regexp, error) {
 // manifestDiff an internal function to compare the manifests difference specified in the input.
 func manifestDiff(aom, bom map[string]*K8sObject, im map[string]string, verbose bool) (string, error) {
 	var sb strings.Builder
+	out := make(map[string]string)
 	for ak, av := range aom {
 		ay, err := av.YAML()
 		if err != nil {
 			return "", err
 		}
 		bo := bom[ak]
 		if bo == nil {
-			writeStringSafe(&sb, "\n\nObject "+ak+" is missing in B:\n\n")
+			out[ak] = fmt.Sprint("\n\nObject " + ak + " is missing in B:\n\n")
```

Comment:
`fmt.Sprint` or string concatenation but not both at the same time, e.g.

```golang
out[ak] = `\n\nObject ` + ak + ` is missing in B:\n\n`
// or
out[ak] = fmt.Sprintf("`\n\nObject %v is missing in B:\n\n`, ak)
```

Reply:
Done, thanks.